src/controlers/usersController.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+const functions = require('../functions/users');
```

Comment:
rename this variable to 'usersService', this is a simple remark but improves the readability and understanding of the code